xen/arch/x86/include/asm/intel_txt.h

```diff
@@ -120,32 +122,16 @@ static inline void txt_reset(uint32_t error)
 	while (1);
 }
 
-/*
- * Secure Launch defined MTRR saving structures
- */
-struct txt_mtrr_pair {
-	uint64_t mtrr_physbase;
-	uint64_t mtrr_physmask;
-} __packed;
-
-struct txt_mtrr_state {
-	uint64_t default_mem_type;
-	uint64_t mtrr_vcnt;
-	struct txt_mtrr_pair mtrr_pair[TXT_OS_MLE_MAX_VARIABLE_MTRRS];
-} __packed;
-
 /*
  * Secure Launch defined OS/MLE TXT Heap table
  */
 struct txt_os_mle_data {
 	uint32_t version;
 	uint32_t boot_params_addr;
-	uint64_t saved_misc_enable_msr;
-	struct txt_mtrr_state saved_bsp_mtrrs;
+    uint32_t slrt;
```

Comment:
Inconsistent indentation, but I've got a feeling that the rest of the file is wrong. Let's leave it in this state, it will have to be changed in existing commit during rebase/squash/upstream.

Reply:
Yeah, almost all other lines are indented with tabs.